.github/workflows/main.yaml

```diff
@@ -267,7 +270,8 @@ jobs:
         shard: ${{ fromJson(needs.initialize.outputs.on_device_test).tests }}
         include: ${{ fromJson(needs.initialize.outputs.includes) }}
     env:
-      COBALT_BOOTLOADER: ${{needs.initialize.outputs.bootloader}}
+      COBALT_BOOTLOADER: ${{ needs.initialize.outputs.bootloader }}
```

Comment:
I thought we had pre-commit yaml formatter enabled by default, no ?

Reply:
It seems it does not format these expressions. @isarkis maybe you know?

Reply:
I don't think our yaml linter can handle GitHub Actions specific yaml syntax.